Tests/Unit/Role-Band-Levels/RoleBandServiceTests.ts

```diff
@@ -0,0 +1,40 @@
+var axios = require('axios');
+var MockAdapter = require('axios-mock-adapter');
+var chai = require('chai');  
+const expect = chai.expect;
+const roleBandService = require('../../../service/RoleBandService');
+const { Console } = require('console');
+const roleBand = {
```

Comment:
This variable should be typed. 

```suggestion
const roleBand: RoleBandResponse = {
```